### DISCUSSION TITLE
Disable spam filtering

### DISCUSSION THREAD

**Discussion by OSXGroupFL** [rspamd/rspamd#4629] [Q&A]
Can someone please provide the settings to disable the spam checking in mailcow. We have a front end filtering service and don't want the overhead on the server AT ALL. I have been googling for almost an hour with ZERO answers, it should not be this hard, it's 2023 and a lot companies are using 3rd party filtering services so being able to turn off the spam checking should be a simple option and for some reason it seems to be a big deal.

**Comment by fatalbanana** [comment]
It's a question for mailcow people I suppose.

You could [configure Rspamd not to filter](https://github.com/rspamd/rspamd/discussions/4628) supposing there is a way to get at Rspamd configuration but you'd be better off not running it & sending it messages in the first place.
